pom.xml

```diff
@@ -308,6 +308,32 @@
         </configuration>
       </plugin>
 
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>flatten-maven-plugin</artifactId>
+        <version>1.0.0</version>
+        <configuration>
+          <flattenMode>ossrh</flattenMode>
```

Comment:
I assume by flattening we would still able to build different interpreter on different scala version in one `mvn package` or `mvn install` command though